Sorting Algorithms/Quick Sort Algorithm/Java/QuickSortDijsktra.java

```diff
@@ -0,0 +1,79 @@
+/* ===== ===== =====
+
+Theory of Programming
+
+Quick sort algorithm with Dijksrta's 3-way paritioning
+http://theoryofprogramming.com/2015/01/20/quick-sort-algorithm/
+GitHub - https://github.com/VamsiSangam/theoryofprogramming
+Code Contributor - Aditya Aggarwal (github.com/the-cybersapien, cybersapien.xyz)
+
+===== ===== ===== */
+
+import java.util.Scanner;
+
+public class QuickSortDijsktra{
+
+	public static void main(String[] args) {
+        //Initialize the Scanner Object to take input
```

Comment:
Please add a space after ```//```, so your comment must go like ```// Initialize the Scanner Object to take input```. I'll update the conventions.